Explicitly start and kill epmd, to avoid lingering instances after tests are
complete.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -276,12 +276,13 @@ test_elixir: test_stdlib test_ex_unit test_logger test
 
 test_stdlib: compile
 	@ echo "==> elixir (ex_unit)"
-	$(Q) exec epmd & exit
+	$(Q) exec epmd${MODERL_VERSION} & exit
 	$(Q) if [ "$(OS)" = "Windows_NT" ]; then \
 		cd lib/elixir && cmd //C call ../../bin/elixir.bat --sname primary -r "test/elixir/test_helper.exs" -pr "test/elixir/**/$(TEST_FILES)"; \
 	else \
 		cd lib/elixir && ../../bin/elixir --sname primary -r "test/elixir/test_helper.exs" -pr "test/elixir/**/$(TEST_FILES)"; \
 	fi
+	$(Q) exec epmd${MODERL_VERSION} -kill
 
 #==> Dialyzer tasks
 
